src/frontend/src/pages/SettingsPage/UserScheduleSettings/UserScheduleSettingsView.tsx

```diff
@@ -6,12 +6,43 @@
 import { Grid } from '@mui/material';
 import DetailDisplay from '../../../components/DetailDisplay';
 import { NERButton } from '../../../components/NERButton';
-import { UserScheduleSettings } from 'shared';
+import { DesignReview, UserScheduleSettings } from 'shared';
 import { useState } from 'react';
 import SingleAvailabilityModal from './Availability/SingleAvailabilityModal';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+import AvailabilityEditModal from './Availability/AvailabilityEditModal';
+import { useMarkUserConfirmed } from '../../../hooks/design-reviews.hooks';
+import { useToast } from '../../../hooks/toasts.hooks';
 
-const UserScheduleSettingsView = ({ scheduleSettings }: { scheduleSettings: UserScheduleSettings }) => {
+const UserScheduleSettingsView = ({
+  scheduleSettings,
+  designReview
+}: {
+  scheduleSettings: UserScheduleSettings;
+  designReview?: DesignReview;
+}) => {
   const [availabilityOpen, setAvailabilityOpen] = useState(false);
+  const toast = useToast();
+  const user = useCurrentUser();
+  const defaultOpen = designReview && !designReview.confirmedMembers.map((user) => user.userId).includes(user.userId);
```

Comment:
This assertion causes the preferences page not to open when there's no design review